File: Fundamentos de Programacion/C/1. Basicos/3. Iteracion/Ejercicio1.c
/*
5) Solicitar un número entero positivo al usuario y calcular su factorial. En el caso de ingresar un número negativo mostrar un mensaje que diga “no se puede calcular el factorial del número ingresado”. Recordar que por definición factorial(0)=1 y factorial(1)=1
*/

# include <stdio.h>

int main(){
    int numero;
    int factorial = 1;

    printf("Ingrese un numero entero positivo: ");
    scanf("%d", &numero);
    if (numero < 0){
        printf("No se puede calcular el factorial del numero ingresado.\n");
    } else {
        for (int i =1; i <= numero; i++){
            factorial *= i;
        }
        printf("El factorial de %d es: %d\n", numero, factorial);
    }
    return 0;
}

File: Algoritmos y Estructura de Datos/Java/src/1. Introducción a Java/A. Estructuras consecutivas o secuenciales/Ejercicio1.java
import java.util.Scanner;

public class Ejercicio1{
    public static void main(String[] args){

        Scanner scanner = new Scanner(System.in);
        
        int numero;
        
        System.out.print("Ingrese un numero entero: ");
        numero = scanner.nextInt();

        System.out.println("Numero ingresado = " + numero);

        scanner.close();
    }
}